### PR TITLE
OSOE-156: Code styling

### PR DESCRIPTION
**Pull request by Piedone**
OSOE-156

### CODE REVIEW

**Comment by 0liver** [comment]
What an adventure!

File: .github/actions/setup-sql-server/Initialize-SqlServer.ps1 [hunk 1]
```diff
@@ -4,12 +4,17 @@ if ($Env:RUNNER_OS -eq "Windows")
 }
 else
 {
+    # For whatever reason, except name, the switches need to only include a single dash despite them needing a double
+    # dash when executing directly from the command line.
```
Comment (0liver) [inline .github/actions/setup-sql-server/Initialize-SqlServer.ps1]: Please remove or rephrase the comment.

**Review by 0liver — CHANGES_REQUESTED** [review]

**Comment by Piedone** [comment]
It was a huge waste of time fighting with the peculiarities of how splatting works.

**Comment by 0liver** [comment]
You might have been able to test that locally - not the docker command, just the argument splatting.

Anyway, you are smarter now! 🎉

It's probably worth sharing this hard-won knowledge in our wiki on some PS specific page.

**Comment by Piedone** [comment]
I did what I can locally, but most of the stuff was related to Docker (under Ubuntu). I'll add this to the wiki.

**Comment by sarahelsaig** [comment]
You don't have to test it with docker tho.
image

**Comment by Piedone** [comment]
That's what I thought, but hash table-based splatting doesn't work with anything non-PS. Array-based splitting does, but I could get that right in one go, after spending half an hour figuring out why hash table-based splatting won't work the same as with a PS script.

**Comment by sarahelsaig** [comment]
You can see from my screenshot that splatting a hash table prepends the keys with a single dash. So I wouldn't expect it to work with `docker` that expects longform arguments to have the GNU style double dashes.

**Comment by Piedone** [comment]
That's what I did, except for `--name` it does need double dashes too. [This code](https://github.com/Lombiq/GitHub-Actions/blob/8022fce306fa8eb51e2b972e2b77e8f72102f230/.github/actions/setup-sql-server/Initialize-SqlServer.ps1) with [this build](https://github.com/Lombiq/Open-Source-Orchard-Core-Extensions/actions/runs/3197976199/jobs/5222126951) is where I had to give up hash tables.

But please let's not waste any more time on this whole thing.

**Comment by sarahelsaig** [comment]
What you linked actually results in `docker run ---name sql2019 --env 'ACCEPT_EULA=Y SA_PASSWORD=Password1!' --publish '1433:1433' --detach 'mcr.microsoft.com/mssql/server:2019-latest'`. Note the triple dash. Like I said, splatting a table prepends one dash to the key. It's a real misstep from Microsoft because if it prepended double dash then it would be more compatible.
image

It appears that you've given up right before removing the final double dash from your hash table.
image

**Review by 0liver — APPROVED** [review]

**Pull request merged**